main/src/core/MonoDevelop.Ide/ExtensionModel/MainMenu.addin.xml

```diff
@@ -241,7 +241,7 @@
 	</ItemSet> 
 
 	<ItemSet id = "Help" _label = "_Help">
-		<CommandItem id = "MonoDevelop.Ide.Commands.HelpCommands.Help" />
+<!--		<CommandItem id = "MonoDevelop.Ide.Commands.HelpCommands.Help" /> -->
```

Comment:
Just a bit confused by this commit compared to the commit message. Is this removing all help, or just Doc Browser stuff?